parm/config/config.aeroanl

```diff
@@ -5,13 +5,20 @@
 
 echo "BEGIN: config.aeroanl"
 
-export OBS_YAML_DIR=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/config/
-export OBS_LIST=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/lists/aero_prototype.yaml
-export AEROVARYAML=$HOMEgfs/sorc/gdas.cd/parm/aero/variational/3dvar_dripcg.yaml
-export BERROR_YAML=$HOMEgfs/sorc/gdas.cd/parm/aero/berror/static_bump.yaml
-export FV3JEDI_FIX=$HOMEgfs/fix/gdas
+export CASE_ANL=${CASE}
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/aero/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/aero/obs/lists/gdas_aero_prototype.yaml
+export AEROVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/variational/3dvar_gfs_aero.yaml
+export STATICB_TYPE='identity'
+export BERROR_YAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/berror/staticb_${STATICB_TYPE}.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
+export BERROR_DATA_DIR=${FV3JEDI_FIX}/bump/aero/${CASE_ANL}/
+export BERROR_DATE="20160630.000000"
 
 export io_layout_x=@IO_LAYOUT_X@
 export io_layout_y=@IO_LAYOUT_Y@
 
+export JEDIVAREXE=${HOMEgfs}/exec/fv3jedi_var.x
+export CRTM_VER="2.3.0"
```

Comment:
Why is this hard-wired here?
Is this going to be a problem when we move to CRTM v2.4.0 this month.

Reply:
This is because the CRTM modulefile used by GSI defines this, but the JEDI bundle does not. This is a temp thing and will need to be rectified at a later date. It is mainly to determine where the coefficients come from in the fix dir (the JEDI and GSI coeffs are not interchangeable due to 'endianness' among other things...)

Reply:
Ok. Thanks for explaining.